manifests/master/codemanager.pp

```diff
@@ -14,6 +14,13 @@
       false => $classroom::params::gitserver['github'],
     }
 
+    if ($use_gitea) and ($control_owner != 'root') {
```

Comment:
I **really** don't like this pattern. I'd much prefer that our *offline* configuration simply required changing `offline => true` rather than a combination of three different variables that need to change.